icu4c/source/common/unicode/platform.h

```diff
@@ -347,6 +347,18 @@
 #   define U_HAVE_INTTYPES_H U_HAVE_STDINT_H
 #endif
 
+/**
+ * \def U_SUPPORTS_ATOMICS
+ * Defines whether the platform supports C++ atomics.
+ * Set this to 0 to disable certain code that requires atomics.
+ * @internal
```

Comment:
Should it be `@internal`? It seems that many of the other flags are `@internal`, even if the have a `U_` prefix (not just `UPRV_`)

Reply:
good idea, especially since it's not used everywhere so you can't build all of ICU with it